pandas/tests/frame/test_timezones.py

```diff
@@ -166,3 +166,19 @@ def test_constructor_data_aware_dtype_naive(self, tz_aware_fixture):
         result = DataFrame({"d": [pd.Timestamp("2019", tz=tz)]}, dtype="datetime64[ns]")
         expected = DataFrame({"d": [pd.Timestamp("2019")]})
         tm.assert_frame_equal(result, expected)
+
+    @pytest.mark.parametrize(
+        "initial",
+        ["2018-10-08 13:36:45+00:00", "2018-10-08 13:36:45+03:00"],  # Non-UTC timezone
+    )
+    @pytest.mark.parametrize("method", ["min", "max"])
+    def test_preserve_timezone(self, initial: str, method):
+        # GH 28552
+        # Given
+        initial_dt = pd.to_datetime(initial)
```

Comment:
can you move to pandas/tests/frame/test_analytics.py (at the very end).

Reply:
Do you mean to have this test as a standalone function, or move it to be a method of `TestDataFrameReductions`?

Reply:
i think as an additional test in TestDataFrameReductions is fine (we are going to split this out at some point anyhow to a new file)

Reply:
Done.